packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -216,6 +171,70 @@ public void onMapReady(GoogleMap googleMap) {
     }
   }
 
+  private static TextureView findTextureView(ViewGroup group) {
```

Comment:
It might be valuable to document that this method returns the first texture view it finds.

Reply:
Done.